src/api/migraphx.py

```diff
@@ -220,6 +220,27 @@ def module(h):
              returns='migraphx::instruction_ref')
 
 
+@auto_handle()
+def value(h):
+    h.constructor('create')
+    h.method('is_null', returns='bool', const=True)
+    h.method('get_key', returns='const std::string', const=True)
+    cpp_types = ['int64_t', 'uint64_t', 'double', 'std::string', 'bool']
+    vt = ['int64', 'uint64', 'float', 'string', 'bool']
+    for vt, cpp_type in zip(vt, cpp_types):
+        if (vt == 'string'):
+            h.constructor('create_' + vt, api.params(i='const char*'))
```

Comment:
I would prefer using str.format() or fstrings here rather than concatenating `vt` with +.

Reply:
The api.py should automatically convert the `std::string` parameter to `const char*`, so I dont think the `if` here is necessary.

Reply:
Addressed both comments. Using format strings and letting auto convert for string.